packages/gatsby-theme-podcast/src/templates/episode-page.js

```diff
@@ -11,6 +12,51 @@ import SEO from "../components/SEO";
 import { rhythm } from "../utils/typography";
 
 import "./player-styles.css";
+import "./transcript.css";
+
+const TranscriptParagraph = props => {
+  let hasTimestamp = false;
+  const children = React.Children.toArray(props.children).map((child, i) => {
+    if (i === 0 && typeof child === "string") {
+      const timestamp = child.match(/^\[(\d\d):(\d\d)(?::(\d\d))?\]/);
+      if (timestamp) {
+
+        let seconds = 0;
+        if (timestamp[3]) {
+          seconds =
+            Number(timestamp[1]) * 3600 +
+            Number(timestamp[2]) * 60 +
+            Number(timestamp[3]);
+        } else {
+          seconds = Number(timestamp[1]) * 60 + Number(timestamp[2]);
+        }
+        hasTimestamp = seconds;
+
+        return (
+          <a
+            key={child}
+            className="timestamp"
+            href={`#playFrom=${seconds}`}
+          >
+            {child}
+          </a>
+        );
+      }
+    }
+    return child;
+  });
+
+  return (
+    <p id={hasTimestamp ? `playFrom=${hasTimestamp}` : ''}>
```

Comment:
I guess if we want it to move to the correct paragraph we need an `id`?? It's covered by the player atm though so it would be nice to move the player to the bottom maybe?

Reply:
Example

![test](https://user-images.githubusercontent.com/588473/62388349-13af7d80-b52b-11e9-9e5a-8319ad17d569.gif)

Reply:
I like the way heavybit does it, where there's no jump, and the player sticks to the top on scroll https://www.heavybit.com/library/podcasts/jamstack-radio/ep-42-structuring-content-with-simen-svale-skogsrud-and-knut-melvaer-of-sanity-io/